src/EA.Weee.Web/Areas/Scheme/Controllers/ManageEvidenceNotesController.cs

```diff
@@ -239,24 +234,55 @@ public async Task<ActionResult> ViewEvidenceNote(Guid pcsId, Guid evidenceNoteId
         }
 
         [HttpGet]
-        public async Task<ActionResult> CreateAndPopulateEvideneSummaryViewModel(Guid pcsId, SchemePublicInfo scheme, DateTime currentDate, 
+        public async Task<ActionResult> CreateAndPopulateEvidenceSummaryViewModel(Guid pcsId, SchemePublicInfo scheme, DateTime currentDate, 
             ManageEvidenceNoteViewModel manageEvidenceNoteViewModel)
         {
             using (var client = apiClient())
             {
                 var complianceYear = SelectedComplianceYear(currentDate, manageEvidenceNoteViewModel);
 
-                var request = new GetObligationSummaryRequest(scheme.SchemeId, complianceYear, false, pcsId);
+                GetObligationSummaryRequest request = null;
+                if (scheme.IsBalancingScheme)
+                {
+                    // PBS do not have a scheme id - we send a null to the Stored Proc which will use organisation id instead
+                    request = new GetObligationSummaryRequest(null, pcsId, complianceYear);
+
+                    var evidenceSummaryData = await client.SendAsync(User.GetAccessToken(), request);
+
+                    var pbsSummaryModel = mapper.Map<SummaryEvidenceViewModel>
+                        (new ViewEvidenceSummaryViewModelMapTransfer(pcsId, evidenceSummaryData, manageEvidenceNoteViewModel, scheme, currentDate, complianceYear));
+
+                    pbsSummaryModel.NumberOfSubmittedNotes = GetSubmittedNotes();  // will be done in another story
+
+                    return View("SummaryEvidencePBS", pbsSummaryModel);
+                }
+
+                // used by Scheme users
+                request = new GetObligationSummaryRequest(scheme.SchemeId, pcsId, complianceYear);
 
                 var obligationEvidenceSummaryData = await client.SendAsync(User.GetAccessToken(), request);
 
                 var summaryModel = mapper.Map<SummaryEvidenceViewModel>
                     (new ViewEvidenceSummaryViewModelMapTransfer(pcsId, obligationEvidenceSummaryData, manageEvidenceNoteViewModel, scheme, currentDate, complianceYear));
 
+                summaryModel.NumberOfSubmittedNotes = GetSubmittedNotes();  // will be done in another story
```

Comment:
@GuidoWorkDev  Remove this from here as it will be moved to the view model creator at some point and not set in the action